docs/cloud-native-security/vuln-management-findings.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[vuln-management-findings]]
+= Findings
+
+The vulnerabilities findings page displays the vulnerabilities detected by the <<vuln-management-overview, Elastic Vulnerability Management for Cloud (EVMC) integration>>. EVMC findings include metadata such as the CVE identifier, CVSS score, severity, affected package, and fix version if available, as well as information about impacted systems.
```

Comment:
Replace Elastic Vulnerability Management for Cloud (EVMC) integration to Cloud native vulnerability management (CNVM) integration.